packages/hyperion-autologging/src/ALSurfaceMutationPublisher.ts

```diff
@@ -94,7 +94,10 @@ export function publish(options: InitOptions): void {
           if (cacheElementReactInfo) {
             const elementInfo = ALElementInfo.getOrCreate(node);
             reactComponentData = elementInfo.getReactComponentData();
-            elementName = getElementName(node);
+            const elementNameResult = getElementName(node);
+            if(elementNameResult) {
+              elementName = elementNameResult.text;
+            }
```

Comment:
we would probably want the source here as well. 
Consider that when we receive them in WWW, we will will need to look at source + FBT and other infor and decide to keep the value or drop/encode it.